### PR TITLE
Add LogBot to the preinstalls

### PR DESCRIPTION
**Pull request by givinghawk**
the bot from https://github.com/givinghawk/LogBot
let me know if i have not done something right

### CODE REVIEW

File: nodejs/discord-bots/discordjs-logbot/index.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    // get the servers id
+    var serverid = message.channel.id;
+    // if the message is not from the bot, add the message to messages.txt
+    fs.appendFile("./logs/" + serverid + '.txt', tag + ': ' + message.content + '\n', (err) => {
+        if (err) throw err;
+    });
+    if(message.author.bot) return;
+    // set the prefix to the prefix in config.json
+    var prefix = config.prefix;
+    // if the message is "stop" and the message authors id is 545636523580850186 then stop the bot
```
Comment (hwalker928) [inline nodejs/discord-bots/discordjs-logbot/index.js:105]: Please update this comment

File: nodejs/discord-bots/discordjs-logbot/index.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+        process.exit();
+    }
+    // if the message is prefix + "download", send ./logs/guildname.txt to hastebin
+    if (message.content == prefix + 'download' && message.member.id == message.guild.ownerId) {
+        // store the last 100 lines of the file
+        var lines = fs.readFileSync("./logs/" + message.guild.id + '.txt').toString().split('\n').slice(-501);
+        // replace every "ॐ" with a newline
+        var logs = lines.join('\n');
+        // send the archive to hastebin
+        fetch('http://132.145.29.171:1035/documents', {
```
Comment (hwalker928) [inline nodejs/discord-bots/discordjs-logbot/index.js:118]: What is this URL?

File: nodejs/discord-bots/discordjs-logbot/maifest.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (hwalker928) [inline nodejs/discord-bots/discordjs-logbot/maifest.json:1]: Misspelt file name

**Review by hwalker928 — COMMENTED** [review]
Thanks for the contribution, here are some comments

File: nodejs/discord-bots/discordjs-logbot/index.js (hunk 2 above)
Comment (givinghawk) [reply nodejs/discord-bots/discordjs-logbot/index.js:118]: a hastebin install, probably will change it in a second to be the web server that the bot makes

Comment (hwalker928) [reply nodejs/discord-bots/discordjs-logbot/index.js:118]: Use https://paste.sparked.host please

File: nodejs/discord-bots/discordjs-logbot/maifest.json (hunk 3 above)
Comment (givinghawk) [reply nodejs/discord-bots/discordjs-logbot/maifest.json]: fixed

File: nodejs/discord-bots/discordjs-logbot/index.js (hunk 1 above)
Comment (givinghawk) [reply nodejs/discord-bots/discordjs-logbot/index.js]: updated

File: nodejs/discord-bots/discordjs-logbot/index.js (hunk 2 above)
Comment (givinghawk) [reply nodejs/discord-bots/discordjs-logbot/index.js]: fixed

File: nodejs/discord-bots/discordjs-logbot/index.js [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+        fs.writeFileSync("./logs/" + message.channel.id + ".txt", JSON.stringify("", null, 4));
+    }
+    checkFileSize("./logs/" + message.channel.id + '.txt');
+    // get user tag
+    const tag = message.author.tag;
+    // make a date variable
+    var date = new Date();
+    // send date to the console
+    // get the servers id
+    var serverid = message.channel.id;
```
[the remark this answers is not in the document]
Comment (givinghawk) [reply nodejs/discord-bots/discordjs-logbot/index.js:97]: its just for a quick change as this is intented to be used on one guild not multiple

[the remark this answers is not in the document]
Comment (hwalker928) [reply nodejs/discord-bots/discordjs-logbot/index.js:97]: Yes but the rest of the code refers to guild ID?

[the remark this answers is not in the document]
Comment (givinghawk) [reply nodejs/discord-bots/discordjs-logbot/index.js]: fixed

**Pull request merged**